tests/unit/test_queries.py

```diff
@@ -65,31 +74,36 @@ def setUp(self):
 
     def test_queries(self):
         """tests that the queries lambda correctly returns the search results"""
-        ## Arrange ##
+        # Arrange
         response_true = [
             {
                 "_index": "test_data",
                 "_type": "_doc",
                 "_id": "1",
-                "_score": 0.2876821,
+                "_score": 0.18232156,
```

Comment:
Interesting, these scores keep changing in updated PRs! I wonder if this is to be expected, or if we can reliably count on these at some point.